src/quo2/components/animated_header_flatlist/style.cljs

```diff
@@ -12,18 +12,19 @@
    :height   view-height})
 
 (defn button-container
-  [position]
+  [{:keys [left right theme]}]
   (merge
```

Comment:
seems like the `merge` is not needed anymore

Reply:
good spot, will remove 👍